pixeltable/env.py

```diff
@@ -140,6 +142,33 @@ def __init__(self) -> None:
         self._current_conn = None
         self._current_session = None
         self._dbms = None
+        self._event_loop = None
+
+    def _init_event_loop(self) -> None:
+        try:
+            # check if we are already in an event loop (eg, Jupyter's); if so, patch it to allow
+            # multiple run_until_complete()
+            running_loop = asyncio.get_running_loop()
+            import nest_asyncio  # type: ignore[import-untyped]
+
+            nest_asyncio.apply()
+            self._event_loop = running_loop
+            print('Patched running loop')
```

Comment:
Is this print statement intentional? If so, use console logger rather than print()